queries.sql

```diff
@@ -8,3 +8,25 @@
  SELECT * FROM animals WHERE neutered = True;
  SELECT * FROM animals WHERE NOT name =  'Gabumon';
  SELECT * FROM animals WHERE weight_kg BETWEEN 10.3 AND 17.4;
+ BEGIN;
+ UPDATE animals SET species = 'unspecified';
+ ROLLBACK;
+ BEGIN;
+ UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+ UPDATE animals SET species = 'pokemon' WHERE species = '';
+ COMMIT;
```

Comment:
- [ ] Just <strong>above</strong> this `COMMIT` on line 17, please verify that the changes were made by entering a command similar to this `SELECT species from animals`. It is one of the project requirements if you read the description.
- [ ] Just <strong>below</strong> this `COMMIT` on line 17, please verify that the changes persisted by entering a command similar to this `SELECT species from animals`. It is one of the project requirements if you read the description.